README.md

```diff
@@ -1,4 +1,4 @@
-﻿<!-- markdownlint-disable-next-line -->
+<!-- markdownlint-disable-next-line -->
```

Comment:
Why is there a change in this line?

Reply:
i accidentally put my name on top of that line before